modules/repository-s3/src/main/java/org/elasticsearch/repositories/s3/S3BlobStore.java

```diff
@@ -105,46 +108,54 @@ class S3BlobStore implements BlobStore {
         this.repositoryMetadata = repositoryMetadata;
         this.threadPool = threadPool;
         this.snapshotExecutor = threadPool.executor(ThreadPool.Names.SNAPSHOT);
+        this.meter = meter;
+
         this.getMetricCollector = new IgnoreNoResponseMetricsCollector() {
             @Override
             public void collectMetrics(Request<?> request) {
                 assert request.getHttpMethod().name().equals("GET");
-                stats.getCount.addAndGet(getRequestCount(request));
+                final long requestCount = getRequestCount(request);
+                stats.getCount.addAndGet(requestCount);
```

Comment:
I do not think this is relevant to the change.

Reply:
You are absolutely right. This PR was created by removing changes from a PR that is further into other changes. I simply missed to revert these. Thanks for catching it!